pkg/handler/handler_test.go

```diff
@@ -229,7 +229,13 @@ func TestConcurrentReadStaleCache(t *testing.T) {
 
 	jsonResults := make(chan []byte)
 	protoResults := make(chan []byte)
+	updateSpec := make(chan struct{})
```

Comment:
nit: rename to something that indicates this is a channel?

Reply:
`specUpdatedChan`? `specUpdated`? I'm trying to be consistent with the two above

Reply:
I like `specUpdatedChan`, yeah I should have added that suffix to those as well